src/resolve/resolved-conf.c

```diff
@@ -599,3 +599,37 @@ int manager_parse_config_file(Manager *m) {
         return 0;
 
 }
+
+int config_parse_dns_cache_size(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Manager *m = ASSERT_PTR(userdata);
+        int ret, val;
+
+        assert(filename);
+        assert(rvalue);
+
+        if (isempty(rvalue)) {
+                m->cache_size = CACHE_MAX_DEFAULT;
+                return 0;
+        }
+
+        ret = safe_atoi(rvalue, &val);
+
```

Comment:
we try to keep function calls and their immediate error handling together. please place empty lines *before* such function calls, and *after* its immediate error handling, but not between function call and its error handling line you are doing here.